app/views/players/show.html.erb

```diff
@@ -123,6 +141,7 @@
   </div><br><br><br>
 <% end %>
 
+
 <% if @player.position == "Tight End" %>
```

Comment:
The hardcoded position strings like '"Tight End"`, `"Wide Receiver"` in this template would be a little more readable and maintainable if they were hidden behind methods on `Player`. 

Consider introducing `postition?` methods that return booleans and using them on lines 145, 117, 89.

```ruby
if @player.tight_end?
  ...

if @player.running_back?
  ...
```